### PR TITLE
feature - items per page component

### PR DESCRIPTION
**Pull request by mattmull82**
Implemented ItemsPerPage component to allow controlling number of items displayed at a time

### CODE REVIEW

File: components/pagination/items-per-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  private inited:boolean = false;
+
+  constructor(public elementRef:ElementRef) {
+    this.config = this.config || itemsPerPageConfig;
+  }
+
+  ngOnInit() {
+    this.classMap = this.elementRef.nativeElement.getAttribute('class') || '';
+
+    this.itemsPerPage = typeof this.itemsPerPage !== 'undefined' ? this.itemsPerPage : itemsPerPageConfig.itemsPerPage;
```
Comment (binarious) [inline components/pagination/items-per-page.component.ts:56]: Shouldn't `itemsPerPageConfig.itemsPerPage` be `this.config.itemsPerPage`?

Comment (mattmull82) [inline components/pagination/items-per-page.component.ts:56]: No, this is pulling the default from the `const itemsPerPageConfig:IItemsPerPageConfig` (around line 11) if no `itemsPerPage` was provided by the client.

Comment (binarious) [inline components/pagination/items-per-page.component.ts:56]: Then why did you add `config`?

mattmull82 notifications@github.com schrieb am Do., 18. Feb. 2016, 18:56:

Comment (mattmull82) [inline components/pagination/items-per-page.component.ts:56]: Good question, I've removed `config` and added documentation to the `readme.md` c6733ec404c937fe64d265d75329afa0c2b42ad7

**Comment by Dinistro** [comment]
@valorkin Any progress on this?

**Comment by valorkin** [comment]
@Dinistro modals -> date picker popup -> all else :)

**Comment by valorkin** [comment]
In general I agreed that this one can be usefull
Plus I will be rewriting 99% of current code to be 
achieve reusable and extensible component architecture
API will be untouched (in most cases :) )

**Comment by Domainv** [comment]
closes as outdated
